src/main/java/katheryne/Command.java

```diff
@@ -16,10 +16,12 @@ public String executeList(TaskList l) {
         return ui.getListMessage(l);
     }
 
-    public String executeMark(String string) throws MissingInformationException, IndexOutOfBoundsException {
+    public String executeMark(String string)
+            throws MissingInformationException, IndexOutOfBoundsException {
```

Comment:
I like how you placed a line break in the method signature to improve readability!